README.md

```diff
@@ -23,7 +47,6 @@ git clone https://github.com/facultyai/faculty-xval.git
 ```
 
 ##### Create an environment
-
 Next, create an environment in your project named `faculty_xval_{USER_NAME}`.
```

Comment:
I'd recommend using `<`  and `>` here rather than `{` and `}`, to make it clearer than `USER_NAME` is a placeholder rather than an environment variable.
```suggestion
Next, create an environment in your project named `faculty_xval_<USER_NAME>`.
```